main/src/ca/uwaterloo/flix/language/dbg/DocAstFormatter.scala

```diff
@@ -22,6 +22,7 @@ import ca.uwaterloo.flix.language.dbg.DocAst.Expr._
 import ca.uwaterloo.flix.language.dbg.DocAst._
 
 import scala.annotation.tailrec
+import scala.collection.immutable.::
```

Comment:
Wait, this should not be necessary?

Reply:
Scala couldn't figure out that this belonged to List and not DocAst

Reply:
Can we rename our Cons to FCons?